smarts/core/tests/test_smarts_memory.py

```diff
@@ -0,0 +1,190 @@
+import gc
+import time
+
+import gym
+from pympler import tracker, muppy, summary, classtracker
```

Comment:
Have `tracker` `summary` & `classtracker` been used?

BTW, move this `from ... import ...` after `import pytest` will look better